test/common/http/conn_manager_impl_fuzz_test.cc

```diff
@@ -482,15 +482,15 @@ class FuzzStream {
             Fuzz::fromHeaders<TestResponseHeaderMapImpl>(response_action.headers()));
         // The client codec will ensure we always have a valid :status.
         // Similarly, local replies should always contain this.
-        uint64_t status;
-        try {
-          status = Utility::getResponseStatus(*headers);
-        } catch (const CodecClientException&) {
+        const auto status = Utility::getResponseStatusOrNullopt(*headers);
+        if (!status.has_value()) {
           headers->setReferenceKey(Headers::get().Status, "200");
```

Comment:
Not your code. But I am curious: why do we set status 200 i.e., (OK) status for case that no status in the headers ? Should we consider this as the error (like 400)?

Reply:
Yeah, on second thought maybe it doesn't matter too much for fuzzer.
Thanks for the info!